packages/eslint-plugin/typings/eslint-rules.d.ts

```diff
@@ -142,6 +142,79 @@ declare module 'eslint/lib/rules/indent' {
   export = rule;
 }
 
+declare module 'eslint/lib/rules/keyword-spacing' {
+  import { TSESLint, TSESTree } from '@typescript-eslint/experimental-utils';
+  import { RuleFunction } from '@typescript-eslint/experimental-utils/dist/ts-eslint';
+
+  type Option = Partial<{
+    before: boolean;
+    after: boolean;
+  }>;
+  type OverrideOptions = Partial<Record<string, Option>>;
+  type RootOption = Option & { overrides?: OverrideOptions };
+  type Options = [RootOption];
```

Comment:
Could simplify this down. There's not enough duplication to bother deduping.

```suggestion
  type Options = [
  	{
	  before?: boolean;
	  after?: boolean;
	  overrides?: Record<string, {
	    before?: boolean;
	    after?: boolean;
	  }>;
	}
  ];
```